core/src/main/scala/com/thenewmotion/mobilityid/checkDigit.scala

```diff
@@ -50,7 +50,7 @@ private[mobilityid] object LookupTables {
     'R' -> 3, 'S' -> 19, 'T' -> 35,
     'U' -> 7, 'V' -> 23, 'W' -> 39,
     'X' -> 11, 'Y' -> 27, 'Z' -> 43
-  ).mapValues(decode)
+  ).map(r => (r._1, decode(r._2)))
```

Comment:
hmm. migration guide said to simply add `.toMap`

Reply:
Adding .toMap works. It prints a deprecation warning, but we can ignore that until we phase out 2.12 support

Reply:
then let's leave it the way you did it

Reply:
I switched it .mapValues().toMap anyway. It describes the intent better